Cargo.toml

```diff
@@ -21,3 +21,4 @@ consolidate-commits = true
 tag-prefix = ""
 
 [patch.crates-io]
```

Comment:
This now _requires_ at least rust 1.75. Would be good to add this to the `Cargo.toml` to avoid confusion. 

https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field

Reply:
Ah this should be added to resolvo as well of course :)